testgrid/config.yaml

```diff
@@ -4275,14 +4275,14 @@ dashboards:
   - name: crio-e2e-fedora
     test_group_name: test_pull_request_crio_e2e_fedora
     open_test_template:
-        url: https://gcsweb-ci.svc.ci.openshift.org/gcs/<gcs_prefix>/<changelist>.txt
+        url: https://gcsweb-ci.svc.ci.openshift.org/gcs/<gcs_prefix>/<changelist>/artifacts/artifacts/junit_01.xml
```

Comment:
do I need to change this back? it looks wrong now right?

Reply:
this entirely depends on what you want clicking through to the test to do, we usually have it go to gubernator /shrug